.github/workflows/sbom-secscan.yaml

```diff
@@ -0,0 +1,61 @@
+name: SBOM and secscan
+
+on:
+  push:
+    tags: [v*]  # when publishing a release
+  pull_request:  # temporarily for testing in this PR
+  workflow_dispatch:
+
+permissions: {}
+
+jobs:
+  scan:
+    strategy:
+      fail-fast: false
+      matrix:
+        manifest: [snap]
+
+    name: SBOM generation
+    runs-on: [self-hosted, self-hosted-linux-amd64-jammy-private-endpoint-medium]
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v5
+        with:
+          persist-credentials: false
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y libapt-pkg-dev
+          sudo apt install -y python3-apt
```

Comment:
You probably copied this from something I wrote 😊 but it seems odd to have `apt-get` and `apt` here, rather than just consistently use `apt`. Also, I'm not sure that `libapt-pkg-dev` is required any more.